src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -52,16 +53,37 @@ public PersonCard(Person person, int displayedIndex) {
         email.setText(person.getEmail().value);
         description.setText(person.getDescription().value);
         String tutorialsText = person.getTutorials().stream()
-            .map(t -> "T" + t.getValue())
-            .sorted()
-            .collect(Collectors.joining(", "));
+                .map(t -> "T" + t.getValue())
+                .sorted()
+                .collect(Collectors.joining(", "));
         tutorials.setText(tutorialsText);
+        int initialHyperlinksCount = socialMediaLinks.getChildren().size(); // Get the initial count
```

Comment:
I think need to add test case for the new changes.

Reply:
Yep done!